halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Node.java

```diff
@@ -486,7 +486,7 @@ public void makeLocalFilesAbsolute(String halconfigPath) {
                               }
                             }
 
-                            if (fPath == null) {
```

Comment:
This fixes the immediate need, however I think we have a deeper problem because `trustStore` _sometimes_ uses a `@LocalFile` and `@SecretFile` field for something that is not a file reference (this star special value). This means potential bugs everywhere files are processed, like here. I don't see an easy solution without having to separate this special value to its own property in the configuration, thereby impacting also Igor.

Can you verify if the following doesn't break having this star value for trustStore?
1. Backups: `hal backup create` and then `hal backup restore...`
2. Any "mutating" command from CLI, for example change the value of a feature flag.

The above perform some file operations on any `@LocalFile` field, besides the actual `hal deploy apply`, which I assume is working.